api/python/mxnet/symbol_creator.py

```diff
@@ -24,18 +24,17 @@ def __init__(self, name):
             the name of the function
         """
         self.name = name
-        use_param = mx_uint()
-        check_call(_LIB.MXSymDescribe(
-            c_str(self.name),
-            ctypes.byref(use_param)))
-        self.use_param = use_param.value
+        self.handle = handle
+        singleton_ = ctypes.c_void_p()
+        check_call(_LIB.MXSymbolGetSingleton(self.handle, ctypes.byref(singleton_)))
+        self.singleton = Symbol(singleton_)
 
     def __call__(self, **kwargs):
         """Invoke creator of symbol by passing kwargs
 
         Parameters
         ----------
-        params : kwargs
+        params : **kwargs
```

Comment:
should be ```***kwargs:``` with no params, see http://sphinxcontrib-napoleon.readthedocs.org/en/latest/example_numpy.html